tvm/op/stack/reverse.go

```diff
@@ -0,0 +1,55 @@
+package stack
+
+import (
+	"fmt"
+
+	"github.com/xssnick/tonutils-go/tvm/cell"
+	"github.com/xssnick/tonutils-go/tvm/op/helpers"
+	"github.com/xssnick/tonutils-go/tvm/vm"
+	"github.com/xssnick/tonutils-go/tvm/vmerr"
+)
+
+func init() {
+	vm.List = append(vm.List, func() vm.OP { return REVERSE(0, 0) })
+}
+
+func REVERSE(i, j uint8) (op *helpers.AdvancedOP) {
+	op = &helpers.AdvancedOP{
+		Action: func(state *vm.State) error {
+			i += 2
```

Comment:
do not change "globals" in action code, because it can be called again, `xi := i + 2` and use xi then